environment.yml

```diff
@@ -0,0 +1,37 @@
+name: nf-core-coproid-1.0dev
+channels:
+  - conda-forge
+  - bioconda
+  - maxibor
+  - etetoolkit
+  - defaults
+dependencies:
+  - bioconda::adapterremoval=2.2.2
+  - bioconda::bcftools=1.9
+  - bioconda::bedtools=2.27.1
+  - bioconda::blast=2.5.0
+  - anaconda::bokeh=1.0.4
+  - bioconda::bowtie2=2.3.5
+  - bioconda::damageprofiler=0.4.6
+  - bioconda::ete3=3.1.1
+  - bioconda::fastqc=0.11.8
+  - bioconda::htslib=1.9
+  - anaconda::jupyter=1.0.0
+  - bioconda::kraken2=2.0.7_beta
+  - conda-forge::notebook=5.7.5
+  - anaconda::nbconvert=5.4.1
+  - bioconda::nextflow=19.01.0
+  - bioconda::pmdtools=0.60 
+  - bioconda::pysam=0.15.2
+  - maxibor::samtools=1.9
+  - conda-forge::scikit-bio=0.5.5
+  - anaconda::scikit-learn=0.20.2
+  - maxibor::sourcepredict=0.32
+  - conda-forge::umap-learn=0.3.7
+  - pip:
```

Comment:
Please use bioconda packages for everything.

Reply:
Done with #7